src/js/view/controls/controlbar.js

```diff
@@ -275,7 +275,15 @@ export default class Controlbar {
         _model.change('fullscreen', this.onFullscreen, this);
         _model.change('streamType', this.onStreamTypeChange, this);
         _model.change('dvrLive', (model, dvrLive) => {
-            toggleClass(this.elements.live.element(), 'jw-dvr-live', dvrLive === false);
+            const element = this.elements.live.element();
+
+            toggleClass(element, 'jw-dvr-live', dvrLive === false);
+
+            if (element.getAttribute('aria-label') === 'Live') {
```

Comment:
this seems like the wrong check since we want to stop using hard-coded aria-lables; we should check instead wether or not we are at the live edge.

Reply:
`dvrLive` bool should indicate wether we are at live edge or not